cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlUtils.java

```diff
@@ -82,6 +82,40 @@ public static String toDateString(long time, String dateFormat, String timeZone)
     return formatter.format(new Date(time));
   }
 
+  /**
+   * Format the duration from double to human readable string.
+   * @param durationMs Duration in milliseconds
+   * @return String representation of duration
+   */
+  public static String toDurationString(double durationMs) {
+    // If the duration is less than one second, represent in milliseconds.
+    if (durationMs < 1000) {
+      return String.format("%.2f milliseconds", durationMs);
+    }
+
+    durationMs = durationMs / 1000;
```

Comment:
Looks like the meaning of `durationMs` changes as we go along the code due to the reassingments. Can we instead represent all comparison constants in `ms` and then just divide the ms value to the relevant conversion value? -- For example, 1 day is `86400000 milliseconds`

Also can we move the constant times to static variables?